easybuild/easyconfigs/i/imkl/imkl-11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25.eb

```diff
@@ -16,15 +16,13 @@ checksums = ['b57ff502b5f97f2f783e4bbda7ce42b3']
 
 dontcreateinstalldir = 'True'
 
-# license file
-import os
-license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = HOME + '/licenses/intel/license.lic'
```

Comment:
What do you mean exactly?

`HOME` is a constant provided by the EasyBuild framework.

If you're referring to using `/` and that not being compatible with Windows, we're a looooong way away from supporting Windows anyway. And probably there the new bash-in-Windows thing is our best option anyway...